src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
I ran some benchmarks comparing changes here with the commit before it and while I'm sure there's a performance penalty, it gets lost in the noise. Both can do about 35k `wrapper.find("foo")` calls per second.

If the performance is equivalent, why break anyone's code at all?

https://github.com/guncha/enzyme/commit/633b0c5a49d47e3aae76670f333a14e5c1884d65